libs/payments/ui/src/lib/server/terms-and-privacy.tsx

```diff
@@ -0,0 +1,110 @@
+import { getBundle } from '@fxa/shared/l10n/fluent';
+import {
+  GenericTermItem,
+  GenericTermsListItem,
+  PaymentProvider,
+  buildFirefoxAccountsTerms,
+  buildPaymentTerms,
+  buildProductTerms,
+} from '../utils/terms-and-privacy';
+import { headers } from 'next/headers';
+import { FluentBundle } from '@fluent/bundle';
+
+const CONTENT_SERVER_URL = 'https://accounts.stage.mozaws.net'; // TODO - Get from config once FXA-7503 lands
+
+type GenericTermsProps = {
+  title: string;
+  titleId: string;
+  titleLocalizationId: string;
+  items: GenericTermsListItem[];
+  l10n: FluentBundle;
+};
+
+function GenericTerms({
+  titleId,
+  title,
+  titleLocalizationId,
+  items,
+  l10n,
+}: GenericTermsProps) {
+  return (
+    <div className="legal-blurb" role="group" aria-labelledby={titleId}>
+      <p className="legal-heading" id={titleId}>
+        {l10n.getMessage(titleLocalizationId)?.value?.toString() || title}
+      </p>
+
+      <p>
+        {items.map((item) => (
+          <span key={`span-${item.key}`} className="mr-3 last:mr-0">
+            <a
```

Comment:
Should this utilize Next's `Link` component instead?

Reply:
Looking at the [Next.js docs](https://nextjs.org/docs/app/api-reference/components/link), the `<Link>` component is primarily useful for local routing. For external links, like these, I'd prefer to keep using standard `<a>` tags. Although I'm happy to discuss more.